src/QsCompiler/TestTargets/Simulation/Target/QirExecution.cs

```diff
@@ -62,31 +62,54 @@ private static unsafe bool TryParseBitcode(string path, out LLVMSharp.Interop.LL
             }
         }
 
+        private static unsafe void ExplicitLibraryLoad(string libraryName)
+        {
+            if (LLVM.LoadLibraryPermanently(libraryName.AsMarshaledString()) != 0)
+            {
+                throw new ExternalException($"Failed explicit load of library '{libraryName}'");
+            }
+        }
+
         public static unsafe void BuildAndRun(string pathToBitcode, params string[] functionNames)
```

Comment:
@swernli Probably worth documenting why the private lookup paths don't automatically work as this shouldn't be needed ordinarily. We see them loaded and then loaded again. There is a lot of magic knowledge here.

Reply:
Unfortunately, yeah. I'll add some explanatory comments.